myo/__init__.py

```diff
@@ -384,7 +387,7 @@ def _(name, *args, **kwargs):
     elif kind == _myo.event_type_t.arm_unsynced:
         result = result and _('on_unsync')
     elif kind == _myo.event_type_t.arm_synced:
-        result = result and _('on_sync')
+        result = result and _('on_sync', event.arm, event.x_direction)
```

Comment:
Thanks! I'm a little worried that code already using `on_sync` would break but I think this is worth doing.